recipes/transonic/meta.yaml

```diff
@@ -1,59 +0,0 @@
-{% set name = "transonic" %}
-{% set version = "0.2.1" %}
-
```

Comment:
please do not touch it,

Reply:
i am not sure what to do... So i close this pull request and try again :)
Thanks @marcelotrevisani for the blas correction